source/browseMode.py

```diff
@@ -1649,10 +1651,15 @@ def _handleScrollTo(self, obj):
 			# However, pretend we handled it, as we don't want it to be passed on to the object either.
 			return True
 
-		try:
-			scrollInfo = self.makeTextInfo(obj)
-		except:
-			return False
+		if isinstance(obj, NVDAObject):
+			try:
+				scrollInfo = self.makeTextInfo(obj)
+			except (NotImplementedError, RuntimeError):
+				return False
+		elif isinstance(obj, textInfos.TextInfo):
+			scrollInfo = obj.copy()
+		else:
+			raise ValueError(f"{obj} is not a supported type")

```

Comment:
Was it already true that `_handleScrollTo` could be called with either a textInfo or an NVDAObject?

Reply:
No, this is new. Previously it could only be passed in an NVDAObject which would then be made into a TextInfo. Now, you can pass in a TextInfo directly.

Reply:
In that case, can this method be kept simple and pass in the object? There doesn't seem to be any need to create the Text info outside?

Reply:
The ActiveTextPositionChanged UI Automation event is fired presumably on the document (sender is the document) and it is passed in a IUIAutomationTextRange (textRange), which is the specific position within the document. 
We create a TextInfo from the IUIAutomationTextRange and use that as scrollInfo in _handleScrollTo. If we just used an NVDAObject this would not be enough as the NVDAObject would be for the document, and not for the specific position within the document. 
So _handleScrollTo was changed to allow passing ina TextInfo directly.

Reply:
Right, I didn't release that we didn't already have an appropriate NVDA object. It would help if `IUIAutomationActiveTextPositionChangedEventHandler_HandleActiveTextPositionChangedEvent` had type annotations.

Reply:
Sorry, I meant `event_UIA_activeTextPositionChanged`, but it would be nice if both had annotations